core/modules/dashboard/includes/block.menus.inc

```diff
@@ -0,0 +1,140 @@
+<?php
+/**
+ * @file
+ *
+ * Dashboard block displaying menus on the site.
+ */
+class DashboardMenuBlock extends Block {
+
+  /**
+   *  Sets block subject on block view.
+   */
+  function getTitle() {
+    return !empty($this->settings['title']) ? check_plain($this->settings['title']) : t('Menus');
+  }
+
+  /**
+   * 'Admin title' callback for the content type.
+   */
+  function getAdminTitle() {
+    return t('Manage Menus');
+  }
+
+  /**
+   * 'Admin info' callback for the content type.
+   */
+  function getAdminPreview() {
+    return t('Links to administer menus.');
```

Comment:
Can we please use "manage" or "configure" instead of "administer"?

Reply:
fixed.

Reply:
Sorry I was not clear, but this is all over the place (like the `.admin-page` links for example). Would it help to flag every instance of this?